flatdata-rs/lib/src/generator.rs

```diff
@@ -66,34 +66,42 @@ pub fn generate(
     let schemas_path = schemas_path.as_ref();
     let out_dir = out_dir.as_ref();
 
-    // create a virtualenv in the target folder
-    eprintln!("creating python virtualenv");
-    let _ = Command::new("python3")
-        .arg("-m")
-        .arg("venv")
-        .arg(out_dir)
-        .spawn()
-        .map_err(GeneratorError::PythonError)?
-        .wait()?;
-
-    // install dependencies
-    let generator_path = if let Ok(path) = env::var("FLATDATA_GENERATOR_PATH") {
-        // we want to rebuild automatically if we edit the generator's code
-        let path = PathBuf::from(path).canonicalize()?;
-        println!("cargo:rerun-if-changed={}", path.display());
-        eprintln!("installing flatdata-generator from source");
-        path
+    let generator_bin = if let Ok(bin_path) = env::var("FLATDATA_GENERATOR_BIN") {
```

Comment:
Can you also update the docs of this function?

Reply:
Hello, i've added some documentation about FLATDATA_GENERATOR_BIN